app/models/question.rb

```diff
@@ -0,0 +1,8 @@
+class Question < ApplicationRecord
+  belongs_to :quiz
+  has_many :options, dependent: :destroy
+
+  validates :content, presence: true
```

Comment:
what is content?

Reply:
Apu, here `content` is the main question.